engine-cockpit/webContent/resources/composite/SslTable.xhtml

```diff
@@ -77,6 +85,10 @@
             id="algorithm" />
         </p:column>
 
+        <p:column headerText="#{ivy.cm.co('/sslTable/PrivateKey')}" rendered="#{cc.attrs.bean.isKeystore()}">
```

Comment:
I like the idea to make the nature of the key visible. But now private-keys are over-emphasized. In 99% percent users have no private keys and therefore have to read and validate that column which gives them no relevant information. 
Can't we make it simpler? E.g. show a special "lock" icon next to the key-name. Or only show this column if there are any private keys?

Reply:
In the keystore there are either private keys or certificates. If you want to use mTLS you need a certificate and a private key. Therefore, only a certificate without a private key does not make sense for mTLS. Most of the time, customers only load the certificate into the key store and have no clue why mTLS is not working. For me best solution is to show a key for private key entries and a cert for cert entries.